gax-grpc/src/test/java/com/google/longrunning/MockOperationsImpl.java

```diff
@@ -130,4 +130,19 @@ public void cancelOperation(
       responseObserver.onError(new IllegalArgumentException("Unrecognized response type"));
     }
   }
+
+  @Override
+  public void waitOperation(
+      WaitOperationRequest request, StreamObserver<Operation> responseObserver) {
+    Object response = responses.remove();
+    if (response instanceof Operation) {
```

Comment:
This is very strange. The response can have multiple types?

Reply:
Copied verbatim from the standard generated GAPIC library for the LRO proto, as are the methods above.